src/main/java/org/broadinstitute/consent/http/service/InstitutionService.java

```diff
@@ -81,13 +80,13 @@ public void deleteInstitutionById(Integer id) {
     institutionDAO.deleteInstitutionById(id);
   }
 
-  public Institution findInstitutionById(Integer id) {
+  public Institution findInstitutionById(Integer id) throws NotFoundException {
```

Comment:
Why did you add this? In general, I don't think it's a good idea to declare unchecked exceptions, unless they're directly thrown in the class, since if at some point this function no longer throws the exception, there's no way for the compiler or an IDE to determine this.

Reply:
Directly thrown in this class.  See `isInstitutionNull` called on line 85 throws a NotFoundException ~line 139.